develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,205 @@
+
+    ```bash
+    {"statusCode":200,"body":"{\"results\":[{\"Hello World\":\"Hello World\"}]}"}
+    ```
+
+## Key Code Snippets
+
+Complete code snippets are available in the [tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
+
+You can refer to the following key code snippets to complete your application development.
+
+### Connect to TiDB
+
+```typescript
+import mysql from 'mysql2';
+
+const pool = mysql.createPool({
+  host,
+  port,
+  user,
+  password,
+  database,
+  ssl: {
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true,
+  },
+  waitForConnections: true,
+  connectionLimit: 1,
+  maxIdle: 1, // max idle connections, the default value is the same as `connectionLimit`
+  idleTimeout: 60000, // idle connections timeout, in milliseconds, the default value 60000
+  queueLimit: 0,
+  enableKeepAlive: true,
+  keepAliveInitialDelay: 0,
+});
+```
+
+### Insert data
+
+```typescript
+const player = ['1', 10, 500];
+await pool.execute(
+  `INSERT INTO player (id, goods, coins) VALUES (${player[0]}, ${player[1]}, ${player[2]})`
+);
+```
+
+Refer to [Insert data](/develop/dev-guide-insert-data.md) for more information.
+
+### Query data
+
+```typescript
+const results = await pool.execute('SELECT count(*) FROM player');
+console.log(results);
+```
+
+Refer to [Query data](/develop/dev-guide-get-data-from-single-table.md) for more information.
+
+### Update data
+
+```typescript
+const player = ['1', 10, 500];
+await pool.execute(
+  `UPDATE player SET goods = goods + ${play[0]}, coins = coins + ${play[1]} WHERE id = ${play[2]}`
+);
+```
+
+Refer to [Update data](/develop/dev-guide-update-data.md) for more information.
+
+### Delete data
+
+```typescript
+await pool.execute('DELETE FROM player WHERE id = 1');
+```
+
+Refer to [Delete data](/develop/dev-guide-delete-data.md) for more information.
+
+## Considerations
+
+- Complete code snippets and how to run them, see [tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
+- The driver is not highly encapsulated, so you will see a lot of SQL statements in the program. Unlike ORM, because there is no data object, the query object of `mysql2` is represented by a object. Although the Node.js driver is convenient, it still requires manual control of the transaction characteristics because it cannot shield the underlying implementation. If there are no scenarios that must use SQL, it is still recommended to use ORM to write programs. This can reduce the coupling of the program.
+- For more information about how to use mysql2, see [mysql2 official documentation](https://github.com/sidorares/node-mysql2).
+- For more information about how to use AWS Lambda Function with TiDB Cloud, see [Designing High-Performance Applications Using Serverless TiDB Cloud and AWS Lambda](https://aws.amazon.com/blogs/apn/designing-high-performance-applications-using-serverless-tidb-cloud-and-aws-lambda/).
```

Comment:
I think It is better to place these resource links in the **What's next** / **Resources** section instead of **Considerations** section.

```suggestion
```